templates/SilverStripe/CMS/Controllers/Includes/CMSMain_PageList_Sidebar.ss

```diff
@@ -0,0 +1,3 @@
+<div class="panel panel--padded cms-content-view cms-tree-view-sidebar" data-pjax-fragment="Content-PageList-Sidebar">
+	<% include SilverStripe\\CMS\\Controllers\\CMSMain_TreeView %>
```

Comment:
Currently only one view allowed for sidebar; See https://github.com/silverstripe/silverstripe-cms/issues/1664 for suggestion to extend this to allow listview.